apps/meteor/app/models/server/models/LivechatRooms.js

```diff
@@ -37,7 +38,7 @@ export class LivechatRooms extends Base {
 
 	findLivechat(filter = {}, offset = 0, limit = 20) {
 		const query = Object.assign(filter, { t: 'l' });
-		return this.find(query, { sort: { ts: -1 }, offset, limit });
+		return this.findPaginated(query, { sort: { ts: -1 }, offset, limit });
```

Comment:
I don't think there is a `findPaginated` in `Base` class.. I actually moved all queries that needed to use `findPaginated` to raw model

Reply:
Weird, this worked 🤔 let me change